scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -636,13 +706,74 @@
                     [delay]
                         time=500
                     [/delay]
+                    {IF_NOT_HAVE_CREATE_LOYAL 4 {ON_DIFFICULTY (Akladian Wiseman) (Akladian Holyman) (Akladian Wonderman)} 39 37}
```

Comment:
Hmm, well based on szopen's previous comments this line is supposed to be very rare in the Akladian army/culture in general. Maybe switch to a different unit line?

Reply:
well I mean there's just the one of it, unlike the Darknites, which have 3 of them... the point is to address player feedback about Uri suddenly becoming "suicidal" and to make it more likely that he survives his charge on Gawen's keep. He ought to have *some* sort of healer, and I highly doubt he'd use a loyalist white mage, or an elvish shaman, or a saurian... maybe a dunefolk (dune herbalist)? Since they're mercenaries in this campaign, and fight for whoever pays them... and if Lorin doesn't have compunctions against hiring them later, I don't think Uri would, either...